CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## [2.1.0] - 2024-02-05
+
+This update should be relatively safe and backwards compatible. It introduces some
+new features with NervesHub including Pre-shared key authentication and file
+upload/download ability through the console channge. If you were previously
+relying on `NervesHubLink.Connection` functions then you will need to review
+and update your code to use `NervesHubLink` connection functions instead.
+
+* Removed
+  * `NervesHubLink.Connection` was removed in favor is using the connection
+    state of the socket instead.
+
+* Added
+  * Use the console channel to save files to the device (#131)
+  * Send a file to an attached NervesHub web console (#130)
+  * Pre-shared key authentication as an alternative to certificate authentication
+
+* Updated
+  * Default SNI from host if none specified
+  * Default to CAStore when no `:ca_certs` are provided
+  * Dependencies:
+    * `castore` 1.0.4 -> 1.0.5
+    * `excoveralls` 0.16.1 -> 0.18.0
+    * `mint` 1.5.1 -> 1.5.2
+    * `nerves_runtime` 0.13.5 -> 0.13.7
+    * `slipstream` 1.1.0 -> 1.1.1
```

Comment:
I don't think it matters much, but I normally don't list `mix.lock` updates since they're not in the hex package.